src/plant_tools.cpp

```diff
@@ -4,27 +4,27 @@
 // [[Rcpp::export]]
 plant::Internals FF16_oderunner_plant_internals(
```

Comment:
FF16_oderunner_plant_internals should become FF16_oderunner_individual_internals

Reply:
And similar below. perhaps search on oderunner_plant_internals